continuous-testing/starter-kits/README.md

```diff
@@ -2,6 +2,350 @@
 
 This page contains starter kit information, which represent templates, code and configuration to help you get started quickly with continuous testing best practices described in this overall guide. Please see categories and links below for details. 
 
+## Static Application Security Testing
+
+This section contains links to sample actions, templates and configurations that analyze and validate code for security flaws and sensitive information. Identifying security vulnerabilities and sensitive data is an [OSS cybersecurity](https://www.cisa.gov/uscert/ncas/alerts/aa22-137a) [best practice](https://appel.nasa.gov/2022/06/30/spotlight-on-lessons-learned-open-source-and-commercial-web-software-vulnerabilities/).  
+
+### Detect Secrets 
+[detect-secrets](https://github.com/Yelp/detect-secrets), an open-source tool employed and recommended by leading technology companies such as [Microsoft](https://microsoft.github.io/code-with-engineering-playbook/continuous-integration/dev-sec-ops/secret-management/recipes/detect-secrets/), [IBM](https://github.com/IBM/detect-secrets), and [Yelp](https://github.com/Yelp/detect-secrets), is instrumental in identifying sensitive information within project files. Its prowess lies in its extensible Python plugin API, which allows custom rules and heuristics to tackle a broad spectrum of secrets. Designed to scan the project's current state rather than the entire git history, 
```

Comment:
My preference would be to remove references to external companies. We are leading on this issue in providing a solution here. Simply stating “leading organizations” would be helpful instead of referencing external private companies. “Detect Secrets is an industry leading tool to identify secure information included in source controlled files.”

Reply:
I’d recommend removing names from Mermaid graphs, also, that might imply a non-existent endorsement relationship or preference.

Reply:
> My preference would be to remove references to external companies. We are leading on this issue in providing a solution here. Simply stating “leading organizations” would be helpful instead of referencing external private companies. “Detect Secrets is an industry leading tool to identify secure information included in source controlled files.”

Personally - I feel the links to Microsoft and IBM examples are useful for context. @jpl-jengelke is there a wording you can suggest that addresses your concern but keeps these useful links available?

Reply:
> My preference would be to remove references to external companies. We are leading on this issue in providing a solution here. Simply stating “leading organizations” would be helpful instead of referencing external private companies. “Detect Secrets is an industry leading tool to identify secure information included in source controlled files.”

I just [updated](https://github.com/perryzjc/slim/commit/de962f1162cc30df7277eff670c49fad8bf4c72b) the documentation about this. Thank you! @jpl-jengelke

Reply:
> > My preference would be to remove references to external companies. We are leading on this issue in providing a solution here. Simply stating “leading organizations” would be helpful instead of referencing external private companies. “Detect Secrets is an industry leading tool to identify secure information included in source controlled files.”
> 
> Personally - I feel the links to Microsoft and IBM examples are useful for context. @jpl-jengelke is there a wording you can suggest that addresses your concern but keeps these useful links available?

I just made [an update](https://github.com/perryzjc/slim/commit/de962f1162cc30df7277eff670c49fad8bf4c72b) to the documentation based on John's suggestion. @jpl-jengelke 
If we have other wording or still want to keep the links, I can update it later. @riverma

Reply:
Looks like @perryzjc made an update to the wording to clarify company citations. LGTM. @jpl-jengelke - we couldn't identify any company references within the Mermaid snippets, so let's leave that for a later discussion / fix and allow @perryzjc to go forth for now.

Reply:
OK, I'll withdraw my comments for now. We should merge this.